library/sonarqube

```diff
@@ -4,17 +4,17 @@ Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
              Pierre Guillot <pierre.guillot@sonarsource.com> (@pierre-guillot-sonarsource),
              Michal Duda <michal.duda@sonarsource.com> (@michal-duda-sonarsource)
 GitRepo: https://github.com/SonarSource/docker-sonarqube.git
-GitFetch: refs/heads/branch-8
-GitCommit: 8ae0fadc72fef64334998e811f1b9cf68a458a2c
+GitFetch: refs/heads/branch-8.1
+GitCommit: 8fb14312c4780e6aca7d95d3fcc5ae3022cb7276
 
-Tags: 7.9.1-community, 7.9-community, latest, lts
+Tags: 7.9.2-community, 7.9-community, latest, lts
 Directory: 7/community
 
-Tags: 8.0-community-beta, 8-community-beta, community-beta
+Tags: 8.1-community-beta, 8-community-beta, community-beta
 Directory: 8/community
 
-Tags: 8.0-developer-beta, 8-developer-beta, developer-beta
+Tags: 8.1-community-beta, 8-developer-beta, developer-beta
```

Comment:
Shouldn't this tag be "8.1-developer-beta" and not "community"?

Reply:
I think this is fixed in a newer PR #7159